percona-monitoring-and-management/secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
This file cannot be created - only install.sh and app.yaml are allowed

Reply:
You can add below in the install.sh 

cat << EOF | kubectl apply -f - 
apiVersion: v1
kind: Secret
metadata:
  name: pmm-secret
  namespace: default
type: Opaque
data:
  PMM_ADMIN_PASSWORD: $PMM_PASSWORD
EOF

Make a note that when you create a secret using a YAML file you need base64 encoded value over there for that you can try this store in a variable and then using is a secret. echo "PMM_PASSWORD" | base64

Reply:
@spron-in

Reply:
@saiyam1814 thanks for the review! PMM_PASSWORD is encoded indeed earlier in the script.

Reply:
@saiyam1814 fixed it :) thanks!

Reply:
I see its removed but you have not added the secret in install.sh? its not needed now?

Reply:
ok i see what you did here, this looks fine!